pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -539,30 +539,66 @@ func validateSoundDevices(field *k8sfield.Path, spec *v1.VirtualMachineInstanceS
 func validateLaunchSecurity(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec, config *virtconfig.ClusterConfig) []metav1.StatusCause {
 	var causes []metav1.StatusCause
 	launchSecurity := spec.Domain.LaunchSecurity
-	if launchSecurity != nil && !config.WorkloadEncryptionSEVEnabled() {
+
+	if launchSecurity == nil {
+		return causes
+	}
+
+	if !config.WorkloadEncryptionSEVEnabled() && !config.WorkloadEncryptionTDXEnabled() {
```

Comment:
There are quite a few validations necessary in addition. If you odn't mind I can try to pus a WIP CL based on yours, containing our validations.

Reply:
Sure, appreciate it! I’m trying to find out all the validations, but didn't get a full picture yet.

Reply:
@rmohr Have you pushed your WIP PR somewhere?